completion/functions/compstyle_zsh-utils_setup

```diff
@@ -0,0 +1,93 @@
+#!/bin/zsh
+
+function compstyle_zsh-utils_help () {
```

Comment:
Help functions should follow the same format as the other functions. I believe it's clarified in CONTRIBUTING

Reply:
Sorry, can you say more? I don't understand what change you're asking for.

Reply:
```suggestion
function compstyle_zsh-utils_help {
```

It's a small change, but the function style should match between this and the setup function.

Reply:
Resolved in rename of zsh-utils to zshzoo.